plans/phase-1/final-cr-definition.md

```diff
@@ -0,0 +1,965 @@
+          resources:
+            requests:
+              cpu: "500m"
+              memory: "256Mi"
+            limits:
+              cpu: "1"
+              memory: "512Mi"
+status:
+  conditions:
+  - type: Available
+    status: "True"
+    lastTransitionTime: "2025-10-21T10:37:00Z"
+    message: "All shards are healthy"
+  shards: 2
+  readyShards: 2
+  ```
+
+  ### Shard CR - Read-Only Child of TableGroup
+
+* This CR is owned by the `TableGroup`.
+* The `Shard` CR owns the resources for the shard (e.g., `MultiPooler` `StatefulSet` and `Postgres` resources).
+
+```yaml
+apiVersion: multigres.com/v1alpha1
+kind: Shard
+metadata:
+  name: "production-db-orders-tg-0"
+  namespace: example
+  creationTimestamp: "2025-10-21T10:35:00Z"
+  generation: 1
+  resourceVersion: "12399"
+  uid: "e5f6a7b8-1234-5678-90ab-f0e1d2c3b4a8"
+  labels:
+    multigres.com/cluster: "example-multigres-cluster"
+    multigres.com/database: "production_db"
+    multigres.com/tablegroup: "orders_tg"
+    multigres.com/shard: "0"
+  ownerReferences:
+  - apiVersion: multigres.com/v1alpha1
+    kind: TableGroup
+    name: "production-db-orders-tg"
+    uid: "d4e5f6a7-1234-5678-90ab-f0e1d2c3b4a7"
+    controller: true
+    blockOwnerDeletion: true
+spec:
+  images:
+    multipooler: "multigres/multigres:latest"
+    postgres: "postgres:15.3"
+  pools:
+    - type: "replica"
+      cell: "us-east-1b"
+      replicas: 2
+      dataVolumeClaimTemplate:
+        accessModes: ["ReadWriteOnce"]
+        resources:
+          requests:
+            storage: "500Gi"
+      postgres:
+        resources:
+          requests:
+            cpu: "4"
+            memory: "8Gi"
+      multipooler:
+        resources:
+          requests:
+            cpu: "1"
+            memory: "512Mi"
+    - type: "readOnly"
+      cell: "us-east-1a"
+      replicas: 1
+      dataVolumeClaimTemplate:
+        accessModes: ["ReadWriteOnce"]
+        resources:
+          requests:
+            storage: "500Gi"
+      postgres:
+        resources:
+          requests:
+            cpu: "2"
+            memory: "4Gi"
+      multipooler:
+        resources:
+          requests:
+            cpu: "500m"
+            memory: "256Mi"
+          limits:
+            cpu: "1"
+            memory: "512Mi"
+status:
+  conditions:
+  - type: Available
+    status: "True"
+    lastTransitionTime: "2025-10-21T10:38:00Z"
+    message: "Shard is healthy and serving"
+  primaryCell: "us-east-1b"
+  totalPods: 3
+  readyPods: 3
+  ```
+
+## Open Issues / Design Questions
+
+This section captures outstanding questions from the initial design phase.
+
+  * Verify that configuration structure of the manifest is true to the way users would use and understand Multigres.
+  * What fields should be defaulted if the user was not providing templates or inline configuration?
+  * Is the proposed `shard`/`cell`/`tablegroup` structure sufficient for a first iteration, or is it overly complex?
+
+
+# Apendix
```

Comment:
Based on KEP template, all of the points below are not appendix. They are an important detail to make a design decision, so I'd remove this headline.
```suggestion
```